hw_diag/utilities/security.py

```diff
@@ -0,0 +1,17 @@
+import gnupg
+
+gpg = gnupg.GPG()
```

Comment:
Perhaps this should be moved inside the function and also should accept a homedir path to make testing easier?

Reply:
After some thoughts, the unit tests are made to use a test key without accepting a custom homedir.
However, accepting a homedir will be useful in case of when I couldn't make testing work otherwise.
Thank you.

Reply:
So in your current unit tests your test key gets added to your system/user-wide GPG store (in ~/.gnupg)? Do you think updating your actual system configs/settings is desirable just when you want to run some tests?

Like if I pull in these changes and run these tests at my end will they also the key to my GPG key store?

For the actual code calling `gnupg.GPG()` without homedir makes sense as we've already imported the key in Dockerfile. However for test cases it does not seem like a great option.

Also the same comment as above that gpg object should be created inside of the `get_verified_json(data)` function.

```python
def get_verified_json(data):
    # This decryption performs a signature validation and separate the payload from the rest of
    # the signature and not doing any real decryption because the message is ent in cleartext.
    gpg = gnupg.GPG()
    decrypted_data = gpg.decrypt(data)
```

Reply:
Make sense.